libs/ui-toolkit/src/components/dialog/dialog.tsx

```diff
@@ -38,7 +38,9 @@ export function Dialog({
           >
             <Icon name="cross" />
           </DialogPrimitives.Close>
-          {title && <h1 className="text-h5 mb-12">{title}</h1>}
+          {title && (
+            <h1 className="text-h5 text-black-95 mt-0 mb-20">{title}</h1>
```

Comment:
Question (non blocking): Are these colors due to the dialog titles not showing? If so I think we need a dark: class here. Also I'd like to get to the bottom of why the colors aren't being inherited properly